packages/app/src/themes/dark_theme.tsx

```diff
@@ -0,0 +1,81 @@
+import { createTheme, SimpleThemeOptions } from '@backstage/theme';
+import { gold, common, myYellow as yellow } from "./colorTypes";
+import { darkPageTheme } from './pageThemes';
+
+interface customThemes {
+  dark: SimpleThemeOptions
+}
+
+const darkTheme: SimpleThemeOptions = {
+  palette: {
+    type: 'dark',
+    background: {
+      default: '#263238',
+      paper: '#323a45', 
+    },
+    status: {
+      ok: "#1DB954",
+      warning: "#FF9800",
+      error: "#E22134",
+      running: "#2E77D0",
+      pending: "#FFED51",
+      aborted: "#757575"
+    },
+    bursts: {
+      fontColor: "#FEFEFE",
+      slackChannelText: "#ddd",
+      backgroundColor: {
+        default: "#7C3699"
+      },
+    },
+    primary: {
+      main: '#94bfa2',
+    },
+    banner: {
+      info: "#2E77D0",
+      error: "#E22134",
+      text: common.white,
+      link: common.black
+    },
+    border: "#E6E6E6",
+    textContrast: common.black,
+    textVerySubtle: "#DDD",
+    textSubtle: "#6E6E6E",
+    highlight: "#FFFBCC",
+    errorBackground: "#FFEBEE",
```

Comment:
```suggestion

	// info boxes
    errorBackground: "#FFEBEE",
```

Could we add this comment to keep it consistent with the light theme.